### PR TITLE
Selenium: improve and rework the 'PullRequestPluginWithForkTest' test

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR improves and rework the **PullRequestPluginWithForkTest** selenium test.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/9155

### CODE REVIEW

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java [hunk 1]
```diff
@@ -68,117 +64,97 @@
...
-    explorer.waitItem(PROJECT_NAME);
-    explorer.openItemByPath(PROJECT_NAME);
-    explorer.openItemByPath(PROJECT_NAME + "/README.md");
+  public void createPullRequest() throws Exception {
+    String projectUrl = testAuxiliaryRepo.getHtmlUrl() + ".git";
+
+    // import project
+    projectExplorer.waitProjectExplorer();
+    menu.runCommand(WORKSPACE, IMPORT_PROJECT);
+    importWidget.waitAndTypeImporterAsGitInfo(projectUrl, PROJECT_NAME);
```
Comment (artaleks9) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java]: I think, should replace the `importWidget.waitAndTypeImporterAsGitInfo()` to the `git.importJavaApp()`

**Comment by dmytro-ndp** [comment]
@SkorikSergey: it's interesting that you didn't use default github user repository. If it means that we can use only one test repository, let's exchange default and auxiliary test users roles.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java [hunk 2]
```diff
@@ -51,58 +47,53 @@
...
-  @Inject(optional = true)
-  @Named("github.username")
-  private String gitHubUsername;
-
-  @Inject(optional = true)
-  @Named("github.password")
-  private String gitHubPassword;
+  @Inject
+  @Named("github.auxiliary.username")
+  private String githubUserName;
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java]: let's name the field as `githubAuxiliaryUserName` to reflect its value

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java [hunk 3]
```diff
@@ -51,58 +47,53 @@
...
 
   @Inject
-  @Named(CheSeleniumSuiteModule.AUXILIARY)
-  private TestGitHubRepository testAuxiliaryRepo;
+  @Named("github.auxiliary.password")
+  private String githubUserPassword;
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java]: let's name the field as `githubAuxiliaryUserPassword` to reflect its value

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**